### PR TITLE
docs: add contributing, code of conduct, codeowners, and readme files

### PR DESCRIPTION
**Pull request by Micahperez2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
...
+# Overview
+This repository contains a workflow that creates a current release tag and then uses the [build-scan-push](https://github.com/liatrio/github-workflows/blob/main/.github/workflows/build-scan-push.yaml) reusable workflow to build the docker image, run a Trivy vulnerability scan, and then push to GHCR.
+
+## What is installed on this image?
+- Version [1.18](https://dl.google.com/go/go1.18.src.tar.gz) of the Go programming language
+- Version [1.2.1](https://releases.hashicorp.com/terraform/1.2.1/) of infrastructure as code tool Terraform
+- Version [0.38.4](https://github.com/gruntwork-io/terragrunt/releases/download/v0.38.4/terragrunt_linux_amd64) of Terraform wrapper Terragrunt 
+
+## What is the image for?
+The image's intendended purpose is to be able to be used as a reusable image to be used as a jenkins agent. By using the installed features the user is able to create jenkins pipeline that can trigger terraform scripts to deploy and update infrastructure. Additionally, Terragrunt gives the ability to use extra tools for Terraform.
```
Comment (austinjw) [inline README.md]: The intended purpose of this image is for it to be used as a Jenkins agent. By using the installed features the user is able to create Jenkins pipelines that can trigger Terraform scripts to deploy and update infrastructure. Additionally, Terragrunt gives the ability to use extra tools for Terraform.

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+## What is required to run this workflow?
+For this workflow to run, the only thing that needs to be input is a user or organizations `GITHUB_TOKEN` into the secrets of the repo.
```
Comment (austinjw) [inline README.md]: For this workflow to run, the only thing that needs to be input is a user or organizations `GITHUB_TOKEN`, which is automatically provided by GitHub Actions.

**Review by austinjw — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,13 @@
...
+# Overview
+This repository contains a workflow that creates a current release tag and then uses the [build-scan-push](https://github.com/liatrio/github-workflows/blob/main/.github/workflows/build-scan-push.yaml) reusable workflow to build the docker image, run a Trivy vulnerability scan, and then push to GHCR.
+
+## What is installed on this image?
+- Version [1.18](https://dl.google.com/go/go1.18.src.tar.gz) of the Go programming language
+- Version [1.2.1](https://releases.hashicorp.com/terraform/1.2.1/) of infrastructure as code tool Terraform
+- Version [0.38.4](https://github.com/gruntwork-io/terragrunt/releases/download/v0.38.4/terragrunt_linux_amd64) of Terraform wrapper Terragrunt 
+
+## What is the image for?
+The intended purpose of this image is for it to be used as a Jenkins agent. By using the installed features the user is able to create Jenkins pipelines that can trigger Terraform scripts to deploy and update infrastructure. Additionally, Terragrunt gives the ability to use extra tools for Terraform.
```
Comment (ChrisSchreiber) [inline README.md]: Lets put his at the top and include some sample code for configuring the agent and using it in the pipeline... for reference https://plugins.jenkins.io/kubernetes/ https://github.com/liatrio/springtrader/blob/solution2/Jenkinsfile

File: README.md [hunk 4]
```diff
@@ -1 +1,13 @@
-Repo for image-builder-terraform
+# Overview
+This repository contains a workflow that creates a current release tag and then uses the [build-scan-push](https://github.com/liatrio/github-workflows/blob/main/.github/workflows/build-scan-push.yaml) reusable workflow to build the docker image, run a Trivy vulnerability scan, and then push to GHCR.
```
Comment (ChrisSchreiber) [inline README.md]: Lets move this down into a sections called Build

**Review by ChrisSchreiber — CHANGES_REQUESTED** [review]

**Review by austinjw — APPROVED** [review]

**Review by ChrisSchreiber — APPROVED** [review]

**Pull request merged**